README.md

```diff
@@ -133,7 +134,7 @@ Once the installation completed the command wil result in following output:
 ### Local and OLM installation type
 If you installed RHOAM locally or via OLM then you can uninstall one of two ways:
 
-- for local installation use the `local-rhaom-operator Namespace`
+- for local installation use the namespace specified in the RHMI CR's spec.namespaceprefix value.
```

Comment:
Not a big deal but this may be easier to read for people who aren't as familiar with kubernetes/openshift

```suggestion
- for local installation use the namespace that the RHMI CR was created in.
```